cmd/test

```diff
@@ -67,15 +67,15 @@ basic() {
     # If there are any items in /adop compose ps...
     if [ "${#AFTER_HEADER}" -gt "0" ] 
     then
-      echo "This test involves running /adop compose down"
-      read -p "Proceed to shut down running containers? (Y/N)   " -n 1 -r
+      echo "This test will unpause any paused LDOP containers and shut them down."
+      read -p "Proceed to shut down running and paused containers? (Y/N)   " -n 1 -r
       echo
       if [[ $REPLY =~ ^[Yy]$ ]]
       then
         echo "...proceeding..."
         echo "...stopping LDOP..."
-        echo "/adop compose down"
-        ${CONF_DIR}/adop compose down
+        ${CONF_DIR}/adop compose unpause 2>/dev/null
```

Comment:
Would like to test for a particular condition to determine whether running a command is necessary as opposed to just forcing it and hiding output. 

For example, parse output to determine whether there are any paused containers, and if so, unpause them. If there are not any paused containers, then there's no need to unpause. Should also allow us to surface output from unpause/down commands which could be useful to either users of this command or logs in a CI/CD system should some unexpected scenario arise.